### PR TITLE
Add dropdown build options step to release checklist

### PR DESCRIPTION
**Pull request by vepadulano**

### CODE REVIEW

**Comment by eguiraud** [comment]
Thanks Vincenzo! I think this info should be somewhere, let's see if Axel agrees that this is the right place or where else it should be :)

File: for_developers/release_checklist/index.md [hunk 1]
```diff
@@ -102,6 +102,10 @@ This assumes you try to create `v6-22-00-patches`, adjust accordingly.
       - Add a 'highlights' section in the generated release page.
       - If this applies, mark the release as `state: latest` and remove the attribute to the one previously holding it (`git grep "state: latest" -- _releases/`)
       - Create a PR against `root-project/web`.
+  1. Update the list of [build options](https://root.cern/install/build_from_source/#all-build-options)
+      - `cd` into main directory of the `root-project/web` repository.
+      - Run `bash .github/scripts/generateBuildOptions.sh v6-22-00-patches`. This creates the file `_includes/build_options_v6-22-00-patches.md`.
```
Comment (Axel-Naumann) [inline for_developers/release_checklist/index.md:107]: We should probable have `generateBuildOptions.sh` and `make-release-page.py` (currently at `_releases/`) in the same directory. If you agree, could you move one next to the other?

File: for_developers/release_checklist/index.md [hunk 2]
```diff
@@ -102,6 +102,10 @@ This assumes you try to create `v6-22-00-patches`, adjust accordingly.
...
+      - Modify the `install/build_from_source.md` file, appending the created file above to the list of build options dropdown items. Look for tags like `<details markdown="1"><summary markdown="span">` and add the file at the end.
```
Comment (Axel-Naumann) [inline for_developers/release_checklist/index.md:108]: As final step, mention `git checkout -b release-v622; git add _includes/build_options_v6-22-00-patches.md install/build_from_source.md; git commit`?

**Review by Axel-Naumann — APPROVED** [review]
LGTM thanks! Please consider the review comments.

**Comment by vepadulano** [comment]
> Maybe it's also worth mentioning in the checklist to update this table (with major releases)?

I see there is already an entry for 6.26, was that done manually or is it already automatised? Anyway, I would leave that for another PR

**Comment by ferdymercury** [comment]
> I see there is already an entry for 6.26, was that done manually or is it already automatised?

Was done manually via https://github.com/root-project/web/pull/765

**Comment by Axel-Naumann** [comment]
Let's get this merged; we'll rely on @couet to follow up on the doc table. Thanks for reminding us, @ferdymercury !

**Pull request merged**